### ISSUE TITLE
Multiple Issues with PowerPoint diagrams (WMFs)

### ISSUE THREAD

**Issue by 1nikolas** [ONLYOFFICE/DesktopEditors#1108]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (DEB package)

### Version information

7.2.1.36

### Expected Behavior

Wmf diagrams should display correctly and not slow down OnlyOffice

### Actual Behavior

1) The diagram does not display correctly (has some weird artifacts). Here is OnlyOffice vs InkScape (opened the wmf file)

![image](https://user-images.githubusercontent.com/30593419/201544863-6a5d3dae-468b-4428-bc55-93d2e8b0b189.png)

2) When printing/saving to pdf, it doesn't display correctly either (some backgrounds are missing)

![image](https://user-images.githubusercontent.com/30593419/201544904-c4faabda-8245-44c7-acf2-b11fa309a589.png)

3) When copying the diagram (as an image) and pasting it, it shows only a part of it

https://user-images.githubusercontent.com/30593419/201545188-2c8adf31-b941-4ce6-a883-bc72d6ab3174.mp4

4) The slide containing this diagram slows down OnlyOffice dramatically

https://user-images.githubusercontent.com/30593419/201545054-ed2d157d-ff2d-41e2-b4c9-42b4c78ed582.mp4

### Reproduction Steps

[Example files](https://github.com/ONLYOFFICE/DesktopEditors/files/9998241/Archive.zip)
The zip file contains both the pptx and the wmf of the diagram shown above (you can also extract it from the pptx). This happens on other diagrams (WMFs) too but this one has the most dramatic effect.

I should also mention that this diagram is made using a really old version of MS Office (and then modified with a newer one, you can look at the created/modify dates). The diagram itself instead of having proper gradients, it has a line/circle for each color of the gradient and this could be the reason it slows down the program a lot.

### Additional information

TLDR: Fix your WMFs 😅

**Comment by 1nikolas** [comment]
Okay I opened the file again and it looks different, still bad..

![image](https://user-images.githubusercontent.com/30593419/201546558-5fd0d2a6-3a38-48f1-97f0-df00aa69af56.png)

**Comment by ShockwaveNN** [comment]
Hi, please split your issue into different ones

It's very hard to track those issue properly because they seems not directly related and may be fixed in different time so this issue became a mess after some time

**Comment by 1nikolas** [comment]
Sounds like the same issue to me but okay I'll split them

**Comment by ShockwaveNN** [comment]
> 
Sounds like the same issue to me but okay I'll split them

Slowdowns are definitely something else from incorrect render
Printing probably too
